Formula/openshift-cli.rb

```diff
@@ -6,9 +6,8 @@ class OpenshiftCli < Formula
 
   stable do
     url "https://github.com/openshift/oc.git",
-        tag:      "openshift-clients-4.6.0-202006250705.p0",
-        revision: "51011e4849252c723b520643d27d3fa164d28c61"
-    version "4.6.0"
+        branch: "release-4.9"
```

Comment:
A branch is a moving target, which makes these builds non-reproducible, which we shouldn't have. Why have upstream stopped tagging releases?

Reply:
Not sure why exactly they stopped tagging releases. I suspect that they are doing global releases, with one version across all repos relevant for OCP4, and reference the latest commit of the branch for that release.

This is an automated issue mentioning which branches are frozen for the "global" release: https://github.com/openshift/oc/issues/180

Nevertheless, I updated the PR to point to the latest commit of the `release-4.9` branch, which is the latest stable branch for OCP4. This makes the builds reproducible.